source/features/pinned-issues-update-time.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import clockIcon from 'octicon/clock.svg';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import { getRepoGQL } from '../libs/utils';
+
+async function init(): Promise<false | void> {
+	for (const pinnedIssue of select.all('.pinned-issue-item')) {
+		const issueNumber = select('.opened-by', pinnedIssue)!.firstChild!.textContent!.replace(/\D/g, '');
+		const lastUpdated = await getLastUpdated(issueNumber);
+		const lastUpdatedTitle = new Intl.DateTimeFormat('default', {
+			month: 'short',
+			day: 'numeric',
+			year: 'numeric',
+			hour: '2-digit',
+			minute: 'numeric',
+			timeZoneName: 'short'
+		}).format(new Date(lastUpdated));
+
+		const bottomBar = select.all(['.open', '.closed'], pinnedIssue)[0].parentElement!;
+
+		bottomBar.append(
+			<span className="d-none d-md-inline">
+				<span className="issue-meta-section ml-2">
+					{clockIcon()} updated <relative-time datetime={lastUpdated} title={lastUpdatedTitle}/>
```

Comment:
No need for `title`
```suggestion
					{clockIcon()} updated <relative-time datetime={lastUpdated} />
```

Reply:
I am following github. They add it...

Reply:
They don't, check the source of the page. It's the custom element itself that adds it, via JS

Reply:
however if i remove it...
```console
Type '{ datetime: string; }' is not assignable to type 'ClassAttributes<HTMLDivElement> & HTMLAttributes<HTMLDivElement> & { datetime: string; title: string; }'.
  Property 'title' is missing in type '{ datetime: string; }' but required in type '{ datetime: string; title: string; }'.ts(2322)
```

Reply:
`mark-unread` made the same feature and the types followed it.

Fixed in #2803

Reply:
Thanks